src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -91,9 +91,11 @@ export function CurrencySearch({
   }, [defaultTokens, debouncedQuery])
 
   const [balances, balancesAreLoading] = useAllTokenBalances()
+  const balancePrices = useTokenPrices(balances)
```

Comment:
maybe a perf hit for the UX since we're waiting for price queries now - we can't parallelize because we need to know which tokens have a balance and need prices